app/Models/V2/AuditableModel.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace App\Models\V2;
+
+interface AuditableModel
+{
+}
```

Comment:
Since `auditStatuses` is being referenced on this interface in `GetPolygonByProjectController`, it should be defined here as well. I sure wish PHP were just a little more type safe 🤷🏻